### PR TITLE
[#778] Add button that opens zoom tab for the entire ramp chart

### LINKED ISSUE #778: Enhance zoom tab
In addition to zooming into details, zoom tab also shows the breakdown the commits, this functionality however, is not encompassed for the whole ramp chart

Let's add a button that opens zoom tab for the entire ramp chart.

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #778 

```
In addition to zooming into a range of ramp slices, the zoom tab also
shows the breakdown of commits that are within the zoom range. This 
functionality however, is not encompassed for the entire ramp chart.

Let's add a button that opens the zoom tab for the entire ramp chart.
```

### CODE REVIEW

**Comment by jamessspanggg** [comment]
The added feature:

enhancezoomtab

**Comment by jamessspanggg** [comment]
> Better wait for #729 to be merged before this, as you are modifying the same part of logic.

Ok noted.

File: frontend/src/index.pug [hunk 1]
```diff
@@ -170,10 +170,15 @@ html
...
+                a(
+                  title="click to view breakdown of commits",
+                  onclick="deactivateAllOverlays()",
+                  v-on:click="openTabZoom(user, filterSinceDate, filterUntilDate)"
+                )
+                  i.summary-chart__title--button.fas.fa-list-ul
               .summary-chart__ramp(
                 onclick="viewClick(event)",
-                v-on:click="openTabZoom(user)"
+                v-on:click="getTabZoomRange(user)"
```
Comment (fzdy1914) [inline frontend/src/index.pug]: Maybe consider to rename it to `getRangeAndOpenTabZoom`? `getTabZoomRange` is not accurate enough.

File: frontend/src/static/js/v_summary.js [hunk 2]
```diff
@@ -552,25 +552,30 @@ window.vSummary = {
       });
     },
 
-    openTabZoom(userOrig) {
+    getRangeAndOpenTabZoom(userOrig) {
```
Comment (eugenepeh) [inline frontend/src/static/js/v_summary.js]: perhaps `openTabZoomSubrange` would be better

File: frontend/src/index.pug [hunk 3]
```diff
@@ -170,10 +170,15 @@ html
                   title="click to view author's repo"
                 )
                   i.summary-chart__title--button.fab.fa-github
-
+                a(
+                  title="click to view breakdown of commits",
+                  onclick="deactivateAllOverlays()",
```
Comment (eugenepeh) [inline frontend/src/index.pug]: just to check, why is this needed?

Comment (jamessspanggg) [reply frontend/src/index.pug]: the `deactivateAllOverlays()` is to remove the any previous existing dotted lines in the ramp charts since this zooms into the entire ramp chart.
